File: 1975-minimum-distance-to-the-target-element/1975-minimum-distance-to-the-target-element.cs
public class Solution {
    public int GetMinDistance(int[] nums, int target, int start) {
        int ans = Int32.MaxValue;
        int n = nums.Length;
        for (var  i = 0; i < n; i++) {
            if (nums[i] == target)
                ans = Math.Min(ans, Math.Abs(i - start));
        }
        return ans;
    }
}

File: 1975-minimum-distance-to-the-target-element/1975-minimum-distance-to-the-target-element.go
func Min (x, y int) int {
    if x < y {
        return x
    }
    return y
}
func getMinDistance(nums []int, target int, start int) int {
    ans := math.MaxInt
    for idx, num := range nums {
        if num == target {
            ans = Min(ans, int(math.Abs(float64(idx - start))))
        }
    }
    return ans
}